docs/integrations/cloud/aws-amplify.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: "AWS Amplify"
+description: "Learn how to sync secrets from Infisical to AWS Amplify."
+---
+
+Prerequisites:
+
+- Set up and add envars to [Infisical Cloud](https://app.infisical.com)
+
+## Sync Secrets On Build
+
+This approach enables the synchronization of secrets from Infisical to AWS Amplify during container build.
+
+<Steps>
+  <Step title="Generate a service token">
+    Go to your project settings in the Infisical dashboard to generate a [service token](/documentation/platform/token). This service token will allow you to authenticate and fetch secrets from Infisical. Once you have created a service token with the required permissions, you’ll need to provide the token to the CLI installed in your Docker container.
+  </Step>
+  <Step title="Set the service token as an Amplify environment variable">
+    ![aws amplify env console](../../images/integrations/aws/integrations-amplify-env-console.png)
+    1. In the Amplify console, choose App Settings, and then select Environment variables.
+    2. In the Environment variables section, select Manage variables.
+    3. Under Variable, enter the key **INFISICAL_TOKEN**. For the value, enter the generated service token from the previous step.
+    4. Click save.
+  </Step>
+  <Step title="Install Infisical CLI to the Amplify build step">
+    In the prebuild phase, add the command in AWS Amplify to install the Infisical CLI.
+
+    ```yaml
+    build:
```

Comment:
They use containers with AWS amplify. Is this approach relevant to them or is this for building for example npm build?

Reply:
It doesn't matter whether containers or not. You cannot modify the start command in amplify.

You need to provide secret from building steps or second